maintenance/aws-janitor/cmd/aws-janitor-boskos/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM gcr.io/k8s-testimages/gcloud-in-go:0.4
```

Comment:
afaik you would not need gcloud here at all? may not even go? Consider choose a minimal base image that can just fit your needs?

Reply:
Hi @krzyzacy,

I literally just copied from https://github.com/kubernetes/test-infra/blob/master/maintenance/aws-janitor/Dockerfile. Can you please explain which one I should use then?

Reply:
depend on what you need here? I suspect a `debian:stretch` may just work?

Reply:
Hi @krzyzacy,

You act like I know much beyond how to say my own name and put on pants in the morning. I just copied from another file :)

But yeah, that should *probably* work. I can verify it easily enough. I'll queue that update to this PR as well.